skimage/feature/blob.py

```diff
@@ -24,50 +23,58 @@ def _blob_overlap(blob1, blob2):
 
     Parameters
     ----------
-    blob1 : sequence
-        A sequence of ``(y,x,sigma)``, where ``x,y`` are coordinates of blob
-        and sigma is the standard deviation of the Gaussian kernel which
-        detected the blob.
-    blob2 : sequence
-        A sequence of ``(y,x,sigma)``, where ``x,y`` are coordinates of blob
-        and sigma is the standard deviation of the Gaussian kernel which
-        detected the blob.
+    blob1 : sequence of arrays
+        A sequence of ``(row, col, sigma)`` or ``(pln, row, col, sigma)``,
+        where ``row, col`` (or ``(pln, row, col, sigma)``) are coordinates
```

Comment:
Or the function should be refactored so that, in ``_prune_blobs``, one would pass only ``blob[:-1]``. What do you think?